app/views/home/index.html.erb

```diff
@@ -1,60 +1,57 @@
 <div id="carouselControls" class="carousel slide" data-ride="carousel">
   <div class="carousel-inner">
     <div class="carousel-item active">
-    <%= image_tag("prueba2.PNG", alt: "prueba", class:"d-block w-100 ") %>
+      <%= image_tag("prueba2.PNG", alt: "prueba", class:"d-block w-100 ") %>
```

Comment:
```suggestion
      <%= image_tag("prueba2.PNG", alt: "prueba", class:"d-block w-100") %>
```

Reply:
hmm 
<img width="1440" alt="Captura de pantalla 2024-05-19 a la(s) 9 21 55 a m" src="https://github.com/NadiaArroyo25/prueba1/assets/130617270/68cbaf0b-f35e-4303-acb1-800c972eaf46">
Should I add another tab? but I think it would be 4 spaces._. I already set them manually in case it was my configuration in visual but it doesn't show any changes